### PR TITLE
[13.0][MIG] website_img_dimension

### PR DESCRIPTION
**Pull request by tarteo**

### CODE REVIEW

**Review by feketemihai — APPROVED** [review]

File: website_img_dimension/static/src/js/website_img_dimension.tooltip.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            $('#wrap').on('mouseenter', 'img', function() {
+                self.show_tooltip($(this));
+            });
+
+            $('#wrap').on('mouseleave', 'img', function() {
+                self.hide_tooltip($(this));
+            });
+        },
+        get_title: function($img) {
+            return $img.width() + ' x ' + $img.height() + ' (' + $img[0].naturalWidth + ' x ' + $img[0].naturalHeight + ')';
```
Comment (Tardo) [inline website_img_dimension/static/src/js/website_img_dimension.tooltip.js:17]:
```suggestion
            return `${$img.width()} x ${$img.height()} (${$img[0].naturalWidth} x ${$img[0].naturalHeight})`;
```

File: website_img_dimension/templates/assets.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Onestein
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <template id="assets_editor" inherit_id="website.assets_editor">
+        <xpath expr="." position="inside">
+            <script
```
Comment (Tardo) [inline website_img_dimension/templates/assets.xml:8]:
```suggestion
            <script type="text/javascript"
```

**Review by Tardo — CHANGES_REQUESTED** [review]
Looks Good. Code review...

**Review by claudiagn — APPROVED** [review]
lgtm!

**Pull request closed without merging**